### PR TITLE
Documentation on uninstallation procedure

### LINKED ISSUE #1733: Uninstall Script
[kind/Feature]

As far as I can tell, there is no uninstall script available for `odo` at this time. It would be helpful to have an OS-independent script similar to the install script. 

## Which functionality do you think we should add?
Uninstall `odo` in an OS-independent manner.

## Why is this needed?
Give users the ability to uninstall `odo`. My use case is that I would like to remove an old version from an environment I am working in.

### PR DESCRIPTION
**Pull request by boczkowska**
## What is the purpose of this change? What does it change?
Added instructions on how to uninstall odo 

## Was the change discussed in an issue?
fixes #1733

### CODE REVIEW

File: docs/installation.adoc [hunk 1]
```diff
@@ -207,3 +207,27 @@ You can edit `Environment Variables` faster using search as follows:
...
+** If you have the scipt locally on your machine:
++
+----
+$ sh installer.sh --uninstall
+----
+
+. Delete local `odo` files: 
++
+----
+$ rm -rf ~/.odo
```
Comment (mik-dass) [inline docs/installation.adoc]: This will remove the `.odo` directory from the home directory only. Maybe add `example` in the above line

Comment (kadel) [reply docs/installation.adoc]: @mik-dass not sure what you mean. It is supposed to remove it from the home directory only.

**Comment by kadel** [comment]
/retest

the error was the same as in https://github.com/openshift/odo/issues/1981

File: docs/installation.adoc (hunk 1 above)
Comment (boczkowska) [reply docs/installation.adoc]: @mik-dass I'm not sure what you mean by adding `example`. The procedure is two-step: 1) uninstall odo w/ script, 2) remove whatever local files are left there in home directory.

Comment (mik-dass) [reply docs/installation.adoc]: @kadel @boczkowska Is this example to remove the odo preference folder? Or is to remove all the `.odo` folder from sub directories? I am kind of confused by the heading `Delete local odo files`

Comment (boczkowska) [reply docs/installation.adoc]: @mik-dass Reworded slightly. Tell me if that makes more sense now. :)

Comment (kadel) [reply docs/installation.adoc]: > @kadel @boczkowska Is this example to remove the odo preference folder? Or is to remove all the `.odo` folder from sub directories? I am kind of confused by the heading `Delete local odo files`

@mik-dass 
odo preference files. Uninstall process shouldn't remove `.odo` folder from the component directories. That is part of the component, and `odo component delete` should handle that.

Comment (mik-dass) [reply docs/installation.adoc]: > odo preference files. Uninstall process shouldn't remove .odo folder from the component directories. That is part of the component, and odo component delete should handle that.

Exactly, that was my concern. Now the rewording seems better.

**Pull request merged**